src/app/my-settings/components/my-settings-notifications/my-settings-notifications.component.html

```diff
@@ -0,0 +1,75 @@
+<div
+  class="erz-container erz-container-limited erz-container-padding-y erz-container-padding-x-responsive"
```

Comment:
Würde der erz-container nicht in die Show-Komponente gehören? Und darin sind dann die einzelnen Settings-Komponenten?

Reply:
Guter Einwand. Hab ich angepasst.